components/package.json

```diff
@@ -5,9 +5,10 @@
     "extends": "enact/strict"
   },
   "dependencies": {
-    "@enact/core": "^3.0.0",
+    "@enact/core": "^4.0.2",
+    "@enact/agate": "enactjs/agate#2.0.0-alpha.3",
```

Comment:
```suggestion
    "@enact/agate": "^2.0.0-beta.1",
```

Reply:
How about using 2.0.0-beta.2 here as it just been released? What do you think?

Reply:
You're right.
Please use 2.0.0-beta.2 instead of beta.1 :)